src/issue.ts

```diff
@@ -1,35 +1,49 @@
-import * as github from '@actions/github';
+import { context } from '@actions/github';
 
 import type { Octokit } from './types';
+import { createContentByDelimiter, swapContentPartiallyByDelimiter } from './text-format';
 
 const ISSUE_TITLE = 'Current Bundle Sizes';
 
-export async function createOrReplaceIssue(
-  octokit: Octokit,
-  body: string,
-): Promise<void> {
-  const { data: issues } = await octokit.rest.issues.listForRepo(github.context.repo);
-
-  const existing = issues.find(issue => issue.title === ISSUE_TITLE);
-
-  if (existing) {
-    const issue_number = existing.number;
-    console.log(`Updating issue ${issue_number} with latest bundle sizes`);
+export async function createOrUpdateIssuePartially({
```

Comment:
My understanding of this is that you'll be spawning many actions for building your project (in a matrix?) and each one will update just its part of the comment?

What are the chances that two run at the same time, and end up overwriting each other? If they both call `issues.listForRepo` before the other one saves, then they'll be operating on stale data.

Reply:
Now that you mention it, I think it's a very real possibility. That hadn't occurred to me.
So for the issues, I can think of a couple of options:
1. Remove support for "Current Bundle Sizes" issues entirely.
2. Create separate issue for every app.
But regardless of the options, I'm not sure how to solve this for PR comments. I'm not sure I want to create a separate comment for every app in the PR as it will create a lot of notification noise.